examples/rpc-examples/rpc-scalajs/project/plugins.sbt

```
@@ -1,4 +1,4 @@
-addSbtPlugin("org.wvlet.airframe" % "sbt-airframe"             % "20.10.1")
-addSbtPlugin("org.scala-js"       % "sbt-scalajs"              % "1.3.1")
+addSbtPlugin("org.wvlet.airframe" % "sbt-airframe" % "21.8.0-2-578ebf23-SNAPSHOT")
```

sbt-airframe 21.8.1 will be released soon
```suggestion
addSbtPlugin("org.wvlet.airframe" % "sbt-airframe" % "21.8.1")
```